vectortile/src/main/scala/geotrellis/vectortile/package.scala

```diff
@@ -0,0 +1,89 @@
+package geotrellis
+
+/** Invented by [[https://www.mapbox.com/ Mapbox]], VectorTiles are a combination of the
```

Comment:
If we attribute Mapbox, can we mention that this is an open specification. Just thinking ahead at having to deal with IP questions about it.